packages/models/src/lib/issue.ts

```diff
@@ -8,7 +8,7 @@ const sourceFileLocationSchema = z.object(
     position: z
       .object(
         {
-          startLine: positiveIntSchema.describe('Start line'),
+          startLine: positiveIntSchema.describe('Start line').optional(),
```

Comment:
The `startLine` can stay required, instead you can take advantage of the fact that the whole `position` object is optional. If `line: 0` is encountered, the `position` should not be set, emittiing `position: { startColumn: 1 }` isn't valid.

```suggestion
          startLine: positiveIntSchema.describe('Start line'),
```